sisyphus/job_path.py

```diff
@@ -403,7 +413,16 @@ def __fs_get__(self, step):
             raise KeyError(step)
 
     def __str__(self):
-        return str(self.get())
+        if gs.FAST_VARIABLE_CONVERSION:
+            return str(self.get())
+        else:
+            value = ''
+            if self.is_set():
+                value = ' %s' % self.get()
+            return "<Variable %s%s>" % (self.rel_path(), value)
 
     def __repr__(self):
-        return str(self.get())
+        if gs.FAST_VARIABLE_CONVERSION:
```

Comment:
thish should be in __str__, but for __str__ we dont really need to change it anyomer (same as Path).